README.md

```diff
@@ -1,4 +1,61 @@
 YDeploy
 =======
 
-REDAXO-Projekte deployen über [deployer](https://deployer.org)
+REDAXO-Projekte deployen über [deployer](https://deployer.org).
+
+Setup für deployer
+------------------
+
+Zunächst sollte man sich mit den Grundlagen von deployer vertraut machen: https://deployer.org
+
+Das Addon liefert deployer selbst nicht mit, es sollte vorzugsweise global mittels Composer installiert werden:
+
+```
+composer global require deployer/deployer
+```
+
+Mehr Infos: https://deployer.org/docs/installation
+
+### Konfiguration
+
+Im Projekt-Root sollte die Konfigurationsdatei `deploy.php`  angelegt werden, die die auf REDAXO abgestimmte [Basis-Konfiguration](https://github.com/yakamara/ydeploy/blob/master/deploy.php) aus diesem Addon einbindet:
+
+```php
+<?php
+
+namespace Deployer;
+
+if ('cli' !== PHP_SAPI) {
+    return;
+}
+
+// Der Pfad ist ggf. anzupassen, falls der Projekt-Root nicht dem REDAXO-Root entspricht
```

Comment:
Unser Hauptgrund für die Struktur war erstmal eine einheitliche Struktur für unsere Frontend-Tools zu haben, egal ob REDAXO oder Symfony. Und das erfüllt die Struktur.
Aber es wäre in der Tat sinnvoll, mal auszutesten, ob man den redaxo-Ordner aus dem Document-Root-Ordner rausschieben kann und wie sich das macht.